src/frontend/components/Footer/Footer.js

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "./Footer.css";
+
+const Footer = () => {
+  return (
+    <footer class="footer">
+      <div class="footer-brand">
+        <Link to="/">
+          <img
+            class="brand-logo"
+            src="assets/Oasis/Logos/oasis-logo-white.svg"
```

Comment:
Again just a suggestion 
you can import and export  all the assets in a  single util file  (just like how  Poornima did it for util functions )